oauth.go

```diff
@@ -262,54 +273,97 @@ func (svc *AlbyOAuthService) AppsShowHandler(c echo.Context) error {
 func (svc *AlbyOAuthService) AppsNewHandler(c echo.Context) error {
 	sess, _ := session.Get("alby_nostr_wallet_connect", c)
 	userID := sess.Values["user_id"]
-	appName := c.QueryParam("c") // c - for client
+	name := c.QueryParam("c") // c - for client
 	if userID == nil {
-		return c.Redirect(302, "/?c="+appName)
+		return c.Redirect(302, "/?c="+name)
+	}
+
+	if name != "" {
+		//auto-create app
+		sk, pk, connectionString, err := svc.CreateApp(name, userID.(uint))
+		if err != nil {
+			svc.Logger.WithFields(logrus.Fields{
+				"pairingPublicKey": pk,
+				"name":             name,
+			}).Errorf("Failed to save app: %v", err)
+			return c.Redirect(302, "/apps")
+		}
+		//check user agent
+		//return page based on user agent
+		if checkMobile(c.Request().UserAgent()) {
```

Comment:
this mobile specific should be done in the client. the website is responsive and can decide on what to show and do. 
hiding and showing things there are easy with CSS.

Reply:
Since this is a server-side rendered app there is less distinction between server and client, I believe this approach is acceptable and easy to read.

Reply:
how is this related to a server-sice renderd app or not? 

it is simple CSS and much less code and duplication.

Reply:
Yeah but I cannot write CSS

Reply:
hehe :D I know some people who can do that.